lib/pdf_info.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+module PdfInfo
+  class MetadataReadError < RuntimeError
+    def initialize(status, stdout)
+      @status = status
+      @stdout = stdout
```

Comment:
what is the purpose of storing these variables? the runtimeerror class doesn't know they exist
if you add `super('error message')` then that is the error message that will show up in the logs, it only supports 1 string though